crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/mod.rs

```diff
@@ -177,7 +177,6 @@ fn execute_syscall<Request: ExecuteCall + SyscallRequest>(
     Ok(())
 }
 
-#[derive(Debug)]
```

Comment:
All `Clone` and `Debug` derives are due to `TraceEntry` not implementing any of these. Maybe I should use the `ProfilerTraceEntry` from here instead of the vm type? It leaks the abstraction, but saves type conversion on potentially thousands of struct entities during `ProfilerCallTrace` building